SEED-platform/seed
# 2476 feat: delete analysis view

#### Any background context you want to provide?
SEED is adding ability to run analyses on properties.

#### What's this PR do?
- refactor bsyncr celery tasks (and task wrapper) to more gracefully handle unexpected issues (e.g. someone deleting the analysis). In addition, avoid as much as possible raising unhandled exceptions that are unnecessary
- add a view for deleting an analysis
- add icon for deleting analysis from the frontend
- fix cases where user message or debug message for AnalysisMessage is too long by truncating the message

#### How should this be manually tested?
Monkey test the analysis functionality, deleting analyses in different statuses and see if you can break anything. After deleting an analysis you should no longer see it.

#### What are the relevant tickets?
#2476 

#### Screenshots (if appropriate)
<img width="1378" alt="Screen Shot 2021-01-05 at 5 10 04 PM" src="https://user-images.githubusercontent.com/18518728/103705362-e6190000-4f78-11eb-8a2e-4b5c6518fc31.png">
